oled-linux.sh

```diff
@@ -84,6 +84,8 @@ do
 
 	step=$((current_brightness - target_brightness))
 	if [ $step -lt 0 ]; then step=$((-step)); fi
+	brightness_step_size=$((step / brightness_step_size_factor))
+	if [ $brightness_step_size -lt $((max_brightness / 500)) ]; then brightness_step_size=$((max_brightness / 500)); fi
```

Comment:
This formula might not be working well for small ranges (i.e. when $max_brightness is 15 -- there are laptops like that)

Reply:
If that really is an issue, I suppose we can change the formula to:
```bash
$((max_brightness / 500 + 1))
```

This will guarantee at least `1` in step size. Furthermore is this really the case with OLED displays though?